src/Core/Prefab/PlanarView.js

```diff
@@ -101,7 +100,7 @@ function PlanarView(viewerDiv, boundingbox, options) {
     tileLayer.update =
         processTiledGeometryNode(
             planarCulling,
-            planarSubdivisionControl(4, SSE_SUBDIVISION_THRESHOLD),
+            planarSubdivisionControl(options.maxSubdivisionLevel || 5),
```

Comment:
`4`  or `5` are both hand-picked values that look good.
Again, we probably want something smarter (e.g depending on the area covered?) but that's for a another PR...